docs/changelog.mdx

```diff
@@ -8,6 +8,34 @@ This is the changelog for the Testkube Control Plane, latest releases are on top
 - **Doc updates** focus on major changes to the documentation
   :::
 
+## Monthly Release v1.19.0 (2025-09-29)
+
+This release is available in Helm Chart 2.325.0.
+
+### What's new!
+
+**Concurrency & Queue Policies.**
+This release is packed with new functionality to throttle and improve the reliability of executions. Define a Workflow Concurrency Policy to control the amount of executions that can run simultaneously across a single or a group of workflows.
+**Runner Concurrency Limits** allow you to prevent overwhelming your runners with too many executions. Whenever a concurrency threshold is met, the execution will be queued until previous executions finish.
+The execution queue might fill up due to these concurrency features or because no online and matching runners can be found. You can now set **Environment Queue Limits** to prevent your workspace from getting overwhelmed.
+ Once this limit is hit, executions will immediately abort which will also trigger its web hooks for alerting.
+ Lastly, **Execution Time-outs** will abort queued and ongoing executions respectively after 1 day and 7 days with further fail-safes when runners are unable to gracefully abort.
+
+**Listener Agents.**
+Now you can listen for TestTrigger events on any Testkube Agent.
```

Comment:
"This release introduces the concept of Listener Agents that allow you to listen for TestTriggers in multiple namespaces/clusters within a single Testkube Environment."